src/panels/lovelace/cards/hui-light-card.ts

```diff
@@ -266,24 +265,21 @@ export class HuiLightCard extends LitElement implements LovelaceCard {
         padding-bottom: 10%;
       }
 
+      .light-button {
+        color: var(--paper-item-icon-color, #44739e);
+        width: 60%;
+        height: auto;
+      }
+
       .slider-center {
         position: absolute;
-        width: 70%;
-        height: 70%;
-        max-height: calc(100% - 40px);
```

Comment:
Does this work with icons of different aspect ratios?

Reply:
Icons are always square